scripts/ci_benchmark.bash

```
@@ -61,7 +61,7 @@ sudo chown -R "$USER:$USER" "$PWD"
 
 rust_bins="$HOME/.docker-cache/cargo/work/release"
 
-mkdir bin
+mkdir -p bin
```

why `-p`, no parents are given.

---

Was failing when running the benchmark repeatedly, -p silently continues if the dir always exists